Update to 1.1.3

scw opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.


<!--
Please add any other relevant info below:
-->
Rebuild to be a `noarch` package, has no compiled components. Add a `pip check` to the tests, this should be sufficient to get this [off the 3.13 migrator list](https://conda-forge.org/status/migration/?name=python313).

scw committed:
Update to the 1.1.3 release

scw committed:
convert to noarch package

scw committed:
add pip check

scw commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.
 * ❌ noarch: python recipes are required to have a lower bound on the python version. Typically this means putting `python >=3.6` in **both** `host` and `run` but you should check upstream for the package's Python compatibility.

For **recipe/meta.yaml**:

 * ℹ️ No valid build backend found for Python recipe for package `wasabi` using `pip`. Python recipes using `pip` need to explicitly specify a build backend in the `host` section. If your recipe has built with only `pip` in the `host` section in the past, you likely should add `setuptools` to the `host` section of your recipe.
 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.
 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13558033602](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13558033602). Examine the logs at this URL for more detail.</sub>

scw committed:
remove noarch

Because of the selector on the `pandoc` piece, treat this as a version
specific package.

Commit:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.45.4, and conda-forge-pinning 2025.02.26.21.58.49

scw committed:
Add 3.13 migration

scw commented:
@conda-forge-admin, please rerender
